segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/WriterState.java

```diff
@@ -71,6 +71,12 @@ class WriterState {
     @GuardedBy("this")
     private ArrayList<ErrorContext> errorContexts;
 
+    /**
+     * Allowed number of events for index segment append
+     */
+    @GuardedBy("this")
+    private long eventSize;
```

Comment:
I think this can be final as it doesn't change

Reply:
Done